index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Porfolio</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+
+        <header>
+
+            <nav>
+        
+                <ul class="navbar">
+                    <li>
+                    <a href="#"> <img src="images/logo.svg" alt="logo"> </a>
+                    </li>
+                    <li>
+                    <a href="#"> <img src="images/menu.svg" alt="menu"> </a>
+                    </li>
+
+                </ul>
+            </nav>
+
+        </header>
+
+        <section>
+            <div class="container">
+                <div class="text">
+                    <h1>I’m Mariane Glad to see you!</h1>
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                </div>
+                <div class="connect">
+                    <p> LET'S CONNECT</p>
```

Comment:
[OPTIONAL] I think it will be better sematically to make this `<p> LET'S CONNECT</p>` in a button tag since it is a call to action and users should be able to click it  `<a href='#><button> LET'S CONNECT</button></a>`